ios/xcode/FluentUITests.xcconfig

```diff
@@ -8,6 +8,7 @@ CODE_SIGN_STYLE = Automatic
 DEVELOPMENT_TEAM = UBF8T346G9
 INFOPLIST_FILE = FluentUI.Tests/Info.plist
 LD_RUNPATH_SEARCH_PATHS = $(inherited) @executable_path/Frameworks @loader_path/Frameworks
+OTHER_LDFLAGS = -ObjC
```

Comment:
Without this flag, our unit tests won't pick up the methods in `FluentTheme+UIKit.swift`. We already use it for building our static lib and application, but were missing it here in the unit tests.